src/main/java/org/diego/tutorial/car/databases/jpa/JPAImplCar.java

```diff
@@ -18,4 +18,16 @@
 		List<Car> carsFromCountry = createQuery.getResultList();
 		return carsFromCountry;
 	}
+
+	/**
+	 * Method that queries the database and retrieves all the non checked cars
+	 * @return List of non checked cars.
+	 */
+	public List<Car> getAllNonCheckedCars() {

```

Comment:
unit test?

Reply:
In the branch of unit tests, they all failed for a reason that I don't know yet, that's why I decided that I should add the unit test later...

Reply:
is unit test branch not merged?

Reply:
Yes, it was merged before I used pull requests.

Reply:
so then everything that gets merged from now on, should not only keep passing unit tests from develop but also add its own tests

Reply:
unit test added for next commit